### DISCUSSION TITLE
Merge your PRs to main 🟢

### DISCUSSION THREAD

**Discussion by ashwinexe** [DiveHQ/backend-internship-task#49] [Q&A]
Hey folks! 
I'm loving your energy to apply to Dive Octernships. 

As a heads up, please **merge the PRs into main branch** in order to allow maintainers review your work. 
Some of you have kept the PRs open or worked on `feedback PR`, which is okay but might cause resistance while evaluating your work. 

Tl;DR Make sure your commits show up on `main` branch

**Comment by swapnalshahil** [comment]
Thanks @ashwinexe for clearing up this confusion.
